### DISCUSSION TITLE
Any plugins with pluginFactory->get_plugin_count > 1

### DISCUSSION THREAD

**Discussion by sagantech** [free-audio/clap#183] [Q&A] [answered]
This condition sort of creates a conundrum for me as my host does not open any files when it scans plug-ins. So to find multiple plug-ins upon instantiation would be something I would need to consider as an alternative. Any one have more than one plug-in contained in a single .clap?

**Comment by baconpaul** [comment, marked as answer]
Sure!

The sample plugins contain 7 but they do need QT to build.

https://github.com/baconpaul/airwin2clap contain 320 or so. I got a report they need a `_USE_MATH_DEFINES` for windows but I think that can be fixed pretty easily. I'll take a swing at that quickly.

https://github.com/surge-synthesizer/tuning-note-claps contains 3 or so and builds easily but are note expressions.

**Reply by abique** [reply]
You can build those plugins headless (no Qt).
u-he zebra2 contains multiple plugins too.

**Reply by sagantech** [reply]
Can someone point me to a binary?

**Reply by baconpaul** [reply]
https://github.com/surge-synthesizer/tuning-note-claps/releases seems i made a binary of the tuning claps. Probably two inside there.

If you don't mind self build, I just confirmed that airwin2clap builds with MSVC and the instructions on the README are up to date.
